load-test-framework/src/main/proto/command.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+package pubsub;
+
+option java_package = "com.google.pubsub.flic.common";
+option java_outer_classname = "Command";
+
+service LoadtestFramework {
+  rpc StartClient(CommandRequest) returns (google.protobuf.Empty) {
+  }
+}
+
+message CommandRequest {
+  google.protobuf.Timestamp start_time = 1;
```

Comment:
If start_time is in the past, we start immediately. If stop_time is in the past, we stop immediately. So if they are both in the past we will basically just immediately start then stop.